compiler/rustc_error_codes/src/error_codes/E0783.md

```diff
@@ -0,0 +1,26 @@
+The range pattern `...` is no longer allowed.
+
+Erroneous code example:
+
+```edition2021,compile_fail,E782
+fn main() {
```

Comment:
I'm curious: why do you need the `main` function here?

Reply:
I mistakenly thought it was necessary